suite-native/module-connect-device/src/components/PinForm.tsx

```diff
@@ -54,8 +73,18 @@ export const PinForm = () => {
                             ))}
                         </HStack>
                     ))}
-                    <PinFormControlButtons />
+                    {!!pinLength && (
+                        <PinFormControlButtons setIsPinFormSubmitting={setIsPinFormSubmitting} />
+                    )}
                 </VStack>
+                {isPinFormSubmitting && (
+                    <VStack style={applyStyle(loaderWrapperStyle)} spacing="medium">
+                        <Loader size="large" />
+                        <Text variant="titleSmall">
+                            {translate('moduleConnectDevice.pinScreen.form.submitting')}
```

Comment:
Please prefer `<Translation>` over `translate()` where easy to use (if you dont need string, but can use reactnode)

Reply:
I also prefer component but translate function is used much more over suite-native (i.e. in this file on line 61). Let's check what others prefer.

Reply:
Now I understand that lot of our components like Button has children type defined as string and that is why so many usage of `translate()`... 
f70898992a25b7a68a604cc9fc2ca5a74483c39d